packages/checkout/shipping/shipping-rates-control/shipping-rates-control.js

```diff
@@ -87,22 +104,54 @@ const ShippingRatesControl = ( {
 			) }
 			showSpinner={ true }
 		>
-			<Packages
+			<ExperimentalOrderShippingPackages.Slot
 				className={ className }
-				collapsible={
-					shippingRates.length > 1 && collapsibleWhenMultiple
-				}
+				collapsibleWhenMultiple={ collapsibleWhenMultiple }
 				noResultsMessage={ noResultsMessage }
 				renderOption={ renderOption }
-				shippingRates={ shippingRates }
+			/>
+			<Packages
+				packages={ shippingRates }
+				noResultsMessage={ noResultsMessage }
 			/>
 		</LoadingMask>
```

Comment:
I'm not sure this code should live in our public package, or if it does make sense to be here. Seeing as this is calling a slotFill, I'd prefer if it's in our internal code.